core/chains/evm/client/node_selector_priority_level.go

```diff
@@ -0,0 +1,103 @@
+package client
+
+import (
+	"math"
+	"sort"
+	"sync/atomic"
+)
+
+type priorityLevelNodeSelector struct {
+	nodes           []Node
+	roundRobinCount []atomic.Uint32
+}
+
+type nodesWithPriority struct {
```

Comment:
nit: should this be `nodeWithPriority` rather than `nodesWithPriority`?

Reply:
Fixed in [`2c3b889` (#9329)](https://github.com/smartcontractkit/chainlink/pull/9329/commits/2c3b889363a9a0274b0163340157234ab4b86162)